dev/com.ibm.ws.microprofile.reactive.streams.operators/resources/META-INF/beans.xml

```diff
@@ -0,0 +1,23 @@
+<!--
+    Copyright (c) 2018 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+   
+    Contributors:
+        IBM Corporation - initial API and implementation
+ -->
+<beans xmlns="http://xmlns.jcp.org/xml/ns/javaee"
```

Comment:
this beans.xml file isn't needed since this bundle isn't a CDI extension

Reply:
Agree- I think that is a copy and paste leftover.